crates/hyperdrive-math/src/long/targeted.rs

```diff
@@ -513,18 +545,19 @@ mod tests {
             // 3. IF Bob's budget is not consumed; then new rate is close to the target rate
 
             // Check that our resulting price is under the max
-            let spot_price_after_long = bob.get_state().await?.calculate_spot_price();
+            let current_state = bob.get_state().await?;
+            let max_spot_price_before_long = current_state.calculate_max_spot_price();
```

Comment:
Either the name or the value is wrong here. This is the max spot price *after* opening the long.